examples/batched-solver/batched-solver.cpp

```diff
@@ -108,46 +150,200 @@ int main(int argc, char *argv[])
     // executor where Ginkgo will perform the computation
     const auto exec = exec_map.at(executor_string)();  // throws if not valid
 
-    auto A = share(gko::read<mtx>(std::ifstream("data/A.mtx"), exec));
-    // Create RHS and initial guess as 1
-    gko::size_type size = A->get_size()[0];
-    auto host_x = vec::create(exec->get_master(), gko::dim<2>(size, 1));
-    for (auto i = 0; i < size; i++) {
-        host_x->at(i, 0) = 1.;
+    const int num_rows = 35;  // per system
```

Comment:
In terms of comments, it might make sense to make use of those `@sect` or any other system to show where the phases of the example start/end (general setup, actually solving the problem, unbatch & data analysis, etc).